addons/packages/packetfence.spec

```diff
@@ -813,6 +815,8 @@ fi
 %config(noreplace)      /usr/local/pf/conf/authentication.conf
 %config(noreplace)      /usr/local/pf/conf/chi.conf
 %config                 /usr/local/pf/conf/chi.conf.defaults
+%config(noreplace)      /usr/local/pf/conf/portal_modules.conf
```

Comment:
From what I saw there is a rule for *.example above that.

I think that's why chi.conf.example isn't there but works.